modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenOperation.java

```diff
@@ -117,7 +117,7 @@ public boolean isRestfulIndex() {
      * @return true if act as Restful show method, false otherwise
      */
     public boolean isRestfulShow() {
-        return "GET".equals(httpMethod) && isMemberPath();
```

Comment:
@sebymiano thanks for the new generator. I noticed that you've removed the condition "isMemberPath". Is that causing issue for the Pistache C++ server generator?

`isRestfulShow` and other methods are used by other server stub generators as well so removing "isMemberPath" may have a wider impact.